src/main/webapp/app/overview/discussion-section/discussion-section.component.ts

```diff
@@ -120,14 +158,17 @@ export class DiscussionSectionComponent extends CourseDiscussionDirective implem
         if (postAVoteEmojiCount < postBVoteEmojiCount) {
             return 1;
         }
-        if (Number(postA.creationDate) > Number(postB.creationDate)) {
-            return 1;
-        }
-        if (Number(postA.creationDate) < Number(postB.creationDate)) {
-            return -1;
+
+        // 5'th criterion
```

Comment:
```suggestion
        // 5th criterion
```
Nitpick (same as above)